dotcom-rendering/src/web/components/LiveBlogEpic.importable.tsx

```diff
@@ -1,43 +1,31 @@
 import { css } from '@emotion/react';
-import { onConsentChange } from '@guardian/consent-management-platform';
-import type { ConsentState } from '@guardian/consent-management-platform/dist/types';
 import type { CountryCode } from '@guardian/libs';
-import { getCookie, joinUrl, log, storage } from '@guardian/libs';
+import { getCookie, log, storage } from '@guardian/libs';
 import { space } from '@guardian/source-foundations';
-import type { ModuleDataResponse } from '@guardian/support-dotcom-components';
+import { getEpicViewLog } from '@guardian/support-dotcom-components';
+import type { EpicPayload } from '@guardian/support-dotcom-components/dist/dotcom/src/types';
 import React, { useEffect, useState } from 'react';
 import { createPortal } from 'react-dom';
+import { useArticleCounts } from '../../lib/articleCount';
 import { submitComponentEvent } from '../browser/ophan/ophan';
 import {
 	getLastOneOffContributionTimestamp,
 	isRecurringContributor,
-	REQUIRED_CONSENTS_FOR_ARTICLE_COUNT,
 	shouldHideSupportMessaging,
+	useConsentForArticleCount,
 } from '../lib/contributions';
 import { getLocaleCode } from '../lib/getCountryCode';
 import { setAutomat } from '../lib/setAutomat';
-import { useApi } from '../lib/useApi';
+import { useSDCLiveblogEpic } from '../lib/useSDC';
 
 type Props = {
 	section: string;
 	shouldHideReaderRevenue: boolean;
 	isPaidContent: boolean;
 	tags: TagType[];
 	contributionsServiceUrl: string;
-};
-
-const useConsent = () => {
```

Comment:
`useConsent` and `consentGivenForArticleCounts` below are replaced with `useConsentForArticleCount`